src/debug.rs

```diff
@@ -4,18 +4,29 @@
 
 use crate::ffi;
 
+/// Default EXI channel. channel can be 0 or 1. Note: Used for device type USBGecko
+pub const DEF_EXICHAN: i32 = ffi::GDBSTUB_DEF_CHANNEL as _;
+
+/// Default TCP port. Note: Used for device type TCP
+pub const DEF_TCPPORT: i32 = ffi::GDBSTUB_DEF_TCPPORT as _;
+
```

Comment:
Its hard to tell that these values can fit into the `channel_port` argument of  `debug_init`. Maybe alude to it in the docs?